.github/workflows/test-staging.yml

```diff
@@ -0,0 +1,18 @@
+name: Test CI pipeline for staging
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build test container
+      run: docker build --target=test -t debricked/debricked-cli-test .
+    - name: Run tests
+      run: docker run -e DEBRICKED_API_URI="$URI" -e DEBRICKED_USERNAME="$USER" -e DEBRICKED_PASSWORD="$PASS" debricked/debricked-cli-test
+      env:
+        USER: ${{ secrets.DEBRICKED_STAGING_USERNAME }}
+        PASS: ${{ secrets.DEBRICKED_STAGING_PASSWORD }}
+        URI: ${{ secrets.DEBRICKED_STAGING_URI }}
```

Comment:
I would like you @zozs to add running tests against prod as well :)

Reply:
You merged the tests against production in May, so they are already in the repo here :stuck_out_tongue: https://github.com/debricked/debricked-cli/blob/master/.github/workflows/test.yml

Reply:
Ah lol. The credentials were not correctly set up though, have done so now :)